### PR TITLE
Removetemplate stacktest

### PR DESCRIPTION
**Pull request by sshveta**
I have commented out a line in request.py.
Please check if that impacts any other test.

I need to check for a test where request is supposed to fail and hence will show status = Error.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 438976c909ecce93d4f96235d8e99aba75e27ec1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9e946166a8594f79e92bb10ddd54c9d58b268a4b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d93f1b40cbb66643448720551c25224742417154:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 472fa7d224e8c6bc109dc34016c25b2bb452e737:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3cdb988391d3d4663272c168683af9be228339e9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4e35b62ba92de512d7c46910da3f2cfe3a32c975:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b5d943576c7c3e2020a60f8e9d9e8df03b62730b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1f7edc692b3262940f076dde0fe31f4529330a0f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 222e0fe2f20927151cb9c8103bbdc42ee1c96fcb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/requests.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def wait_for_request(cells, partial_check=False):
         # Request not found at all, can't continue
         return False
 
-    assert row.status.text != 'Error'
+    # assert row.status.text != 'Error'
```
Comment (seandst) [inline cfme/services/requests.py]: This should probably just get deleted, without a comment. Someone (me, if nobody else beats me to it) needs to look at tests that call this method and make sure we don't have some that are expected an AssertionError to be raised here.

**Comment by dajoRH** [comment]
Lint report for commit 341a3e066eb2ad60026a135e269a93ca9c04fee0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dd323fc1bccc5cedfe953365b0908bac3b5666a9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f20edc916ff36c09b4200a9ba50e655ec1103bf9:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**